### DISCUSSION TITLE
cfdisk "bootable" option not present

### DISCUSSION THREAD

**Discussion by pneumus** [util-linux/util-linux#2220] [General]
Key "B" does not make a partition bootable.
The option [Bootable] is not present in the bottom menu.
Why:?

**Comment by karelzak** [comment]
What label type? I see [Bootable] for MBR (dos), but for example, GPT does not implement it. 

The help is generic, it seems we need to add a note that [Bootable] is implemented for MBR and SGI only.

**Comment by pneumus** [comment]
Actually cfdisk used to ask me in the beginning if I want DOS or GPT but
now this selector did not appear for me

I usually use DOS, that’s why I found it unusual that the bootable option
was not present

Thank you!

**Comment by karelzak** [comment]
I have added a note to the help (commit 793e8d2ab81324d0a712132b26ee16c08575101b).
